README.md

```diff
@@ -1 +1,147 @@
-# portfolio
\ No newline at end of file
+# Portfolio
+
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+      - [Tech Stack](#tech-stack)
+      - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+This is my personal portfolio page design.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- This is a sample of portfolio design.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+This project does not have any live demo still.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy of this repository up and running follow these steps
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. Have a Browser like Chrome, Firefox
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run This command
+
+```sh
+  cd my-folder
+  git clone https://github.com/Esmatullah12/portfolio.git
+```
+
+
+### Install
+
+
+Run This command
+
+```sh
+  cd portfolio
+  gem install
```

Comment:
[OPTINAL] Please change this `gem` to `npm` because in your case you are not using a `gem` so kindly replace it.